File: laravel-demo/app/Http/Controllers/Pages/ProjectsController.php
<?php

namespace App\Http\Controllers\Pages;

use App\User;
use App\Http\Controllers\Controller;

class ProjectsController extends Controller
{
    public function item()
    {
        return view('pages.projects.item', ['title' => 'Project item - Pages']);
    }

    public function list()
    {
        return view('pages.projects.list', ['title' => 'Project list - Pages']);
    }

}

File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/Pages/ProjectsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers.Pages
{
    public class ProjectsController : Controller
    {

         public ActionResult Item()
        {
            return View("~/Views/Pages/Projects/Item.cshtml");
        }

         public ActionResult List()
        {
            return View("~/Views/Pages/Projects/List.cshtml");
        }

    }
}
